test/components/accordions.test.js

```diff
@@ -0,0 +1,49 @@
+const host = `http://localhost:${process.env.JEST_PORT}`;
+
+beforeEach(async () => page.goto(`${host}/components/accordions/`));
+
+test('accordions are open only when aria-expanded="true"', async () => {
+  await expect(page).not.toMatchElement('.test-visible[hidden]');
```

Comment:
for context, should it be asserted that `aria-expanded` is not `true`? Also, why not just use the id selectors as in the subsequent tests? If you're checking `aria-expanded` which s scoped to each accordion container, it may be clearer.

Reply:
> for context, should it be asserted that `aria-expanded` is not `true`?

Great idea! Added in 063a2d1, and did a bit of test refactoring to reduce duplicating that everywhere. 

> Also, why not just use the id selectors as in the subsequent tests?

I was doing so to reduce the number of assertions, but that’s silly (asserting `.test-hidden[hidden]` is actually acting on 2 IDs). I dropped the test classes and asserted the IDs individually in 063a2d1.